doc/rtd/topics/datasources/openstack.rst

```diff
@@ -19,7 +19,7 @@ checks the following environment attributes as a potential OpenStack platform:
 
    * **/proc/1/environ**: Nova-lxd contains *product_name=OpenStack Nova*
    * **DMI product_name**: Either *Openstack Nova* or *OpenStack Compute*
-   * **DMI chassis_asset_tag** is *OpenTelekomCloud* or *OpenStack Nova*
+   * **DMI chassis_asset_tag** is *OpenTelekomCloud*, *SAP CCloud VM* or *OpenStack Nova*
```

Comment:
We generally keep the docs wrapped at 80 chars, could you wrap this line, please?